features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2

```diff
@@ -1106,6 +1106,7 @@
                 <Silver>2000</Silver>
                 <Bronze>1000</Bronze>
                 <Unauthenticated>60</Unauthenticated>
+                <Default_Subscriptionless>100000</Default_Subscriptionless>
```

Comment:
Shall we change this to the naming format used in other elements
```suggestion
                <DefaultSubscriptionless>100000</DefaultSubscriptionless>
```

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/12539/commits/9cd326adf51117359068bd6457efdaf33d209aea